rpi-update

```diff
@@ -3,6 +3,14 @@
 set -o nounset
 set -o errexit
 
+for i in $*; do
+  if [[ $i. == -* ]]; then
+    echo "rpi-update does not command line dash parameters."
```

Comment:
"accept"? "recognise"?

Reply:
accept was the word I was thinking of - updated.